mws/apis/easyship.py

```diff
@@ -0,0 +1,90 @@
+"""
+Amazon Easyship API
+"""
+from __future__ import absolute_import
+
+from ..mws import MWS
+from .. import utils
+
+
+class EasyShip(MWS):
+    """
+        Amazon EasyShip API
+        docs: http://docs.developer.amazonservices.com/en_IN/easy_ship/EasyShip_Overview.html
+    """
+    URI = "/EasyShip/2018-09-01"
+    VERSION = "2018-09-01"
+    NAMESPACE = '{https://mws.amazonservices.in/EasyShip/2018-09-01}'
+
+    def get_pickup_slots(self, marketplace_id=None, amazon_order_id=None, package_width=0,
+                         package_height=0, package_length=0, package_dimensions_uom='cm',
+                         package_weight=0, package_weight_uom='g'):
+        data = {
+            'Action': 'ListPickupSlots',
+            'MarketplaceId': marketplace_id,
+            'AmazonOrderId': amazon_order_id,
+            'PackageDimensions.Width': str(package_width),
+            'PackageDimensions.Length': str(package_length),
+            'PackageDimensions.Height': str(package_height),
+            'PackageDimensions.Unit': package_dimensions_uom,
+            'PackageWeight.Unit': package_weight_uom,
+            'PackageWeight.Value': str(package_weight)
+        }
+        return self.make_request(data)
+
+    def create_scheduled_package(self, marketplace_id=None, amazon_order_id=None, package_width=0,
+                                 package_height=0, package_length=0, package_dimensions_uom='cm',
+                                 package_weight=0, package_weight_uom='g',
+                                 package_slot_id=None, package_slot_start_time=None,
+                                 package_slot_end_time=None, package_identifier=None):
```

Comment:
Related to comment for the `update_` method, please update some of the argument names for consistency: `slot_id`, `slot_start_time`, and `slot_end_time` (I think using `package_` prefix in that case is a little too verbose for our needs).